packages/mdc-list/constants.ts

```diff
@@ -40,19 +40,21 @@ const strings = {
   ARIA_ORIENTATION_HORIZONTAL: 'horizontal',
   ARIA_ROLE_CHECKBOX_SELECTOR: '[role="checkbox"]',
   ARIA_SELECTED: 'aria-selected',
-  CHECKBOX_RADIO_SELECTOR: 'input[type="checkbox"]:not(:disabled), input[type="radio"]:not(:disabled)',
-  CHECKBOX_SELECTOR: 'input[type="checkbox"]:not(:disabled)',
+  CHECKBOX_RADIO_SELECTOR: 'input[type="checkbox"], input[type="radio"]',
+  CHECKBOX_SELECTOR: 'input[type="checkbox"]',
   CHILD_ELEMENTS_TO_TOGGLE_TABINDEX: `
     .${cssClasses.LIST_ITEM_CLASS} button:not(:disabled),
     .${cssClasses.LIST_ITEM_CLASS} a
   `,
+  ENABLED_CHECKBOX_RADIO_SELECTOR: 'input[type="checkbox"]:not(:disabled), input[type="radio"]:not(:disabled)',
+  ENABLED_CHECKBOX_SELECTOR: 'input[type="checkbox"]:not(:disabled)',
```

Comment:
These two ENABLED_* constants are not used anywhere at this point. They are essentially renamed versions of the old `CHECKBOX_RADIO_SELECTOR` and `CHECKBOX_SELECTOR` before this PR, which were too restrictive in terms of what constituted a valid checkbox/radio to be toggled programmatically. Per Matt's philosophy that all programmatic changes to disabled checkboxes are allowed, I think we can remove this restriction and its associated constants?

Reply:
We can remove if unused.